### ISSUE TITLE
Requirement: Combine and drop unused / underspecified bit flags

### ISSUE THREAD

**Issue by krischer** [FDSN/miniSEED3-TechnicalEvaluation#23]
Simplification of MiniSEED2: Combine and drop unused / underspecified bit flags.

**Comment by crotwell** [comment]
Provide standard mapping for any that are deemed important to the addition/optional header structure.

**Comment by krischer** [comment]
### Summary

(Please let me know if I missed a point or misunderstood something)

Please vote on:

1. Should unused/underspecified bit-flags be dropped? (Yes/No)

**Comment by kaestli** [comment]
do we need to agree on what is "widely unused" or "underspecified"?
imho, i have seen little recent usage of everything except the a) time correction flag, b) the leap second flags and c) the the clock lock flag (however for a and b better solutions are available), 
and i think everything is underspecified except the time correction and the leap second flags (thus, the clock lock flag is actually underspecified too)

**Comment by krischer** [comment]
I'd say no as I think most of us agree on which fields are actually used. Furthermore it is a detail that can be figured out in the final stage with a fully specified format.

**Comment by crotwell** [comment]
yes

**Comment by chad-earthscope** [comment]
1. Yes.

The ones I would like to drop are:
* I/O flags, Bit 0: station volume parity error possibly present 
* I/O flags, Bit 1: long record read
* I/O flags, Bit 2: short record read
* I/O flags, Bit 3: start of time series
* I/O flags, Bit 4: end of time series

But to be honest all of them can be retained if we have optional header capability.  With some marked as deprecated if we wish to discourage their use.  The ones above are just the most worthless of the flags and we could drop them to ease the coding to the new standard a bit.

**Comment by kaestli** [comment]
yes

**Comment by ozym** [comment]
1. Yes

**Comment by ketchum-usgs** [comment]
Yes, Chad's list is a good starting list.  I suspect that "clock locked" is also useless and that this should be communicated with a better value.

**Comment by claudiodsf** [comment]
Yes

**Comment by ihenson-bsl** [comment]
1. Yes

**Comment by ValleeMartin** [comment]
1. Yes

**Comment by JoseAntonioJara** [comment]
1. Yes
